Form/Handler.php

```diff
@@ -193,7 +193,8 @@ protected function sendMails(
                 $type->getNotifyFromMailAddress($form->getData()),
                 true,
                 $type->getNotifyReplyToMailAddress($form->getData()),
-                $this->attachments
+                $this->attachments,
```

Comment:
$type->getNotifySendAttachments($form->getData()) ? $this->attachments : array()